app/assets/javascripts/components/projects/ProjectCard.js

```diff
@@ -6,9 +6,11 @@ export default class ProjectCard extends React.Component {
 
     if (joined) {
       return(
-        <div className="panel-heading">
-          <a href={ project.get('path_to').project } className="card-title">{ project.get('name') }</a>
-
+        <div className="panel-heading card-heading">
+          <div>
+            <a href={ project.get('path_to').project } className="card-title project-title">{ project.get('name') }</a>
+            { project.get('tag_name') && <small className="card-tag" style={{ backgroundColor: project.get('tag_bg_color'), color: project.get('tag_fore_color') }}>{project.get('tag_name') }</small> }
```

Comment:
@andrerpbts it's mostly ok to do it like this, but since the line is too long here, you could create a method that returns that and does the conditional internally, like:

```javascript
renderTag(project) {
   if(project.get('tag_name')) {
      return <small ...
   }
```

And use it like this:

```javascript
<div>
   <a ...
   { this.renderTag(project) }
</div>
```

What do you think @brunohkbx